nat-lab/tests/test_features_builder.py

```diff
@@ -87,7 +88,10 @@ def test_telio_features_builder_all_defaults():
         "flush_events_on_stop_timeout_seconds": 0,
         "multicast": true,
         "ipv6": true,
-        "nicknames": true
+        "nicknames": true,
+        "batching": {
+            "direct_connection_threshold": 0
```

Comment:
But this will basicaly don nothing?

Reply:
Yes, this is intentional. The idea is that the batcher touches imtimate parts of libtelio because of keep-alives. Meaning to reduce the friction we basically put it under a feature flag and just care if it doesn't break anything by just being there first. Then we care if it actually works to some degree which can be then fine-tuned through the feature flag :)

TLDR: so far we only care that batcher's existence doesn't break anything.